### PR TITLE
This fixes #12

### LINKED ISSUE #12: Update nextflow_schema.json to match workflow inputs
[nextflow_schema.json](https://github.com/nf-core/pangenome/blob/dev/nextflow_schema.json) does not currently match actual workflow inputs

### PR DESCRIPTION
**Pull request by subwaystation**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/pangenome/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/pangenome _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

Bring the `nextflow_schema.json` up to date fixing #12.

### CODE REVIEW

File: nextflow_schema.json [hunk 1]
```diff
@@ -9,22 +9,21 @@
             "title": "Input/output options",
             "type": "object",
             "fa_icon": "fas fa-terminal",
-            "description": "Define where the pipeline should find input data and save output data.",
+            "description": "Define where the pipeline should find input data and save output. data.",
```
Comment (heuermh) [inline nextflow_schema.json:12]: Minor typo here, `save output. data.` &rarr; `save output data.`

File: nextflow_schema.json [hunk 2]
```diff
@@ -38,42 +37,173 @@
...
-                    "help_text": "If you have no genome reference available, the pipeline can build one using a FASTA file. This requires additional time and resources, so it's better to use a pre-build index if possible."
+                    "default": "10,100,1000,10000",
+                    "description": "Consensus graph specification: write the consensus graph to                                       BASENAME.cons_[spec].gfa; where each spec contains at least a min_len parameter                                       (which defines the length of divergences from consensus paths to preserve in the                                       output), optionally a file containing reference paths to preserve in the output,                                       a flag (y/n) indicating whether we should also use the POA consensus paths, a                                       minimum coverage of consensus paths to retain (min_cov), and a maximum allele                                       length (max_len, defaults to 1e6); implies -a; example:                                       cons,100,1000:refs1.txt:n,1000:refs2.txt:y:2.3:1000000,10000.",
```
Comment (heuermh) [inline nextflow_schema.json:167]: Some extra internal whitespace here

**Review by heuermh — APPROVED** [review]
LGTM, found a couple of minor style issues

**Comment by heuermh** [comment]
Note this lint check is expected to fail
```
╭──────────────────────────────────────────────────────────────────────────────╮
│ [✗] 1 Test Failed                                                            │
├──────────────────────────────────────────────────────────────────────────────┤
│ conda_dockerfile: Could not find Dockerfile file string: FROM nfcore/base:1… │
╰──────────────────────────────────────────────────────────────────────────────╯
```

File: nextflow_schema.json (hunk 1 above)
Comment (subwaystation) [reply nextflow_schema.json:12]: > LGTM, found a couple of minor style issues

Thanks for you quick review!

I don't understand your suggestion here. In the orignial, I already have `save output data.`.

Comment (subwaystation) [reply nextflow_schema.json:12]: I am confused, why your suggestions are in red. Or your references. Anyhow, you are right.

**Review by heuermh — APPROVED** [review]

**Pull request merged**